acos_client/v30/slb/virtual_server.py

```diff
@@ -37,7 +37,7 @@ def _set(self, name, ip_address=None, arp_disable=False, description=None, vrid=
         params = {
             "virtual-server": self.minimal_dict({
                 "name": name,
-                "ip-address": ip_address,
+                #"ip-address": ip_address,
```

Comment:
please help remove the commented code
Thanks

Reply:
done